script.js

```diff
@@ -1,12 +1,194 @@
-//You can edit ALL of the code here
-function setup() {
+// Loads page.
+let setup = () => {
   const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
-}
+  renderHeader();
+  renderEpisodeCards(allEpisodes);
+  renderFooter();
+};
 
-function makePageForEpisodes(episodeList) {
+// Render header elements and adds search box event listener.
+let renderHeader = () => {
+  const allEpisodes = getAllEpisodes();
+
+  let header = document.getElementById("header");
+  let episodeTitleContainer = document.createElement("div");
```

Comment:
I like that your html creation is clean like this, throughout, including well named variables and consistent createElement, appendChild, set properties, that makes it very readable